src/scss/components/footer-block/footer-block.stories.ts

```diff
@@ -0,0 +1,49 @@
+import type { Meta, StoryObj } from "@storybook/web-components";
+
+import { html } from "lit";
+
+const meta: Meta = {
+  title: "Components/Footer/Footer Block",
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+export const UsefulLinks: Story = {
+  render: () => html`<div class="iati-footer-block">
+    <h2 class="iati-footer-block__title">
+      Useful links
+    </h2>
+    <div class="iati-footer-block__content">
+      <ul>
+        <li><a href="#">Useful link</a></li>
```

Comment:
Can we make it more obvious that these are links?

Reply:
@tillywoodfield an underline is the best I could suggest :)

Reply:
I think that would be good

Reply:
Done @tillywoodfield